.github/workflows/tests.yml

```diff
@@ -77,7 +77,6 @@ jobs:
           php-ini-values: "zend.assertions=1"
 
       - name: "Run PHPUnit"
-        run: "vendor/bin/phpunit -v"
+        run: "vendor/bin/phpunit"
```

Comment:
I was thinking about removing the output of skipped tests, so I agree with this change 👍

Reply:
It's also, the `-v` option isn't available on PHPUnit 10